courses/free-courses-hi.md

```diff
@@ -119,7 +119,9 @@
 
 ### HTML and CSS
 
+* [CSS Mastery in one Video 9 hours](https://www.youtube.com/watch?v=WyxzAU3p8CE&t=6063s) - Vishwajeet Kumar(Youtube)
```

Comment:
- Using titles as same as the provided by the original resource
- Removing start timestamp from YT url
- Use syntax for instructors
- No YT annotations

```suggestion
* [CSS Tutorial in Hindi \| Complete CSS Course For Beginners to Advanced \| Step By Step Tutorial](https://www.youtube.com/watch?v=WyxzAU3p8CE) - Tech Gun, `tch.:` Vishwajeet Kumar
```